src/main/java/gregtech/common/tools/ToolSaw.java

```diff
@@ -89,6 +89,11 @@ public void convertBlockDrops(World world, BlockPos blockPos, IBlockState blockS
         return SAW_TOOL_CLASSES;
     }
 
+    @Override
+    public boolean canPlayBreakingSound(ItemStack stack, IBlockState state) {
+        return canMineBlock(state, stack);
```

Comment:
The saw should not play sounds when mining. It is indented for crafting and the axe serves a better mining purpose instead. Additionally harvesting ice with it is extremely loud with sounds as it mines instantly.